### PR TITLE
fix resource mappings

### PR DESCRIPTION
**Pull request by asiekierka**
Please take care of this before #507 - essentially, this one is a lot easier to review (and names some things better, IMO) while *also* containing a lot less changes alongside Enigma's automatic upgrade-related fixes.

### CODE REVIEW

**Review by liach — APPROVED** [review]

**Review by modmuss50 — APPROVED** [review]

File: mappings/net/minecraft/resource/ObjectPassingResourceReloadListener.mapping [hunk 1]
```diff
@@ -0,0 +1,3 @@
+CLASS wv net/minecraft/resource/ObjectPassingResourceReloadListener
+	METHOD a apply (Ljava/lang/Object;Lwt;Lafm;)V
```
Comment (Gegy) [inline mappings/net/minecraft/resource/ObjectPassingResourceReloadListener.mapping]: Not sure about `apply` here, imo `accept` works better. Java stdlib has `Consumer#accept` and `Function#apply`, of which `accept` matches better.

File: mappings/net/minecraft/resource/ObjectPassingResourceReloadListener.mapping [hunk 2]
```diff
@@ -0,0 +1,3 @@
+CLASS wv net/minecraft/resource/ObjectPassingResourceReloadListener
```
Comment (Gegy) [inline mappings/net/minecraft/resource/ObjectPassingResourceReloadListener.mapping]: This name is quite verbose. I believe the term `supplier` fits pretty well in this context; could it be used in the name somehow?

Comment (asiekierka) [reply mappings/net/minecraft/resource/ObjectPassingResourceReloadListener.mapping]: I'm not sure. SupplyingResourceReloadListener?

File: mappings/net/minecraft/resource/ObjectPassingResourceReloadListener.mapping (hunk 1 above)
Comment (asiekierka) [reply mappings/net/minecraft/resource/ObjectPassingResourceReloadListener.mapping]: I'm referring to the "load" and "apply" stages of resource reloading. The canon names are "prepare" and "apply", FWIW.

Comment (asiekierka) [reply mappings/net/minecraft/resource/ObjectPassingResourceReloadListener.mapping]: But we can discuss "load" -> "prepare" later.

**Pull request merged**